### ISSUE TITLE
SignatureTest: Broken pipe

### ISSUE THREAD

**Issue by g4s8** [zerocracy/farm#900]
`SignatureTest` can fail with `IOException` (probably after #893 PR), see build logs http://www.rultor.com/t/14253-378938198
```
java.io.UncheckedIOException: java.io.IOException: Broken pipe
        at org.cactoos.scalar.UncheckedScalar.value(UncheckedScalar.java:60)
        at org.cactoos.scalar.NumberEnvelope.intValue(NumberEnvelope.java:98)
        at com.zerocracy.tools.Signature.asString(Signature.java:102)
        at com.zerocracy.tools.SignatureTest.rendersSignature(SignatureTest.java:36)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
        at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
        at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
        at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
        at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
        at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
        at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
        at org.apache.maven.surefire.junitcore.pc.Scheduler$1.run(Scheduler.java:393)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.IOException: Broken pipe
        at java.io.FileOutputStream.writeBytes(Native Method)
        at java.io.FileOutputStream.write(FileOutputStream.java:326)
        at java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:82)
        at java.io.BufferedOutputStream.flush(BufferedOutputStream.java:140)
        at java.io.FilterOutputStream.close(FilterOutputStream.java:158)
        at org.cactoos.io.TeeInputStream.close(TeeInputStream.java:99)
        at org.cactoos.io.LengthOf.lambda$new$0(LengthOf.java:81)
        at org.cactoos.scalar.NumberEnvelope.lambda$new$1(NumberEnvelope.java:73)
        at org.cactoos.scalar.IoCheckedScalar.value(IoCheckedScalar.java:70)
        at org.cactoos.scalar.UncheckedScalar.value(UncheckedScalar.java:58)
```

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@0crat assign @carlosmiranda

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by carlosmiranda** [comment]
@g4s8 see PR #900 please

**Comment by carlosmiranda** [comment]
@0crat wait for PR #900 merge

**Comment by carlosmiranda** [comment]
@0crat wait for merge of PR #912.

**Comment by 0crat** [comment]
> @0crat wait for merge of PR #912. ([here](https://github.com/zerocracy/farm/issues/900#issuecomment-379949625))

@carlosmiranda Job [#900](https://github.com/zerocracy/farm/issues/900) is already on hold

**Comment by carlosmiranda** [comment]
@g4s8 fixed in #912, can we close this now?

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#900](https://github.com/zerocracy/farm/issues/900) is now out of scope

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)
